source/Betting_Class.py

```diff
@@ -0,0 +1,28 @@
+class betting:
+    def __init__(self, playerList):
+        self.pot=0
+        self.bet=0
+        self.list=playerList
+
+    def square(self):
+        square=True
+        for i in self.list:
+            if i.bet == self.bet:
+                square=True
+            else:
+                square=False
+        return square
+    
+    def make_bet(self, amount: int):
+        valid=False
+        if amount == self.bet:
```

Comment:
Is this like a call? I'm confused as to why you need both this and the >= if statement. 

I may be missing something but let me know!